<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogInWindow</class>
 <widget class="QMainWindow" name="LogInWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LogInWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>80</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_mail">
       <property name="text">
        <string>邮    箱</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_pwd">
       <property name="text">
        <string>密    码</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_mail"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_pwd">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="text">
     <string>登   录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reg">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>注    册</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log_in">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>登    录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modify">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>忘记密码</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_info"/>
    <addaction name="action_advice"/>
    <addaction name="separator"/>
    <addaction name="action_rules"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_advice">
   <property name="text">
    <string>反馈信息</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于我们</string>
   </property>
  </action>
  <action name="action_rules">
   <property name="text">
    <string>游戏规则</string>
   </property>
  </action>
  <action name="action_info">
   <property name="text">
    <string>个人信息</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
